tool_path_planner/src/raster_tool_path_planner.cpp

```diff
@@ -30,19 +29,90 @@
 #include <vtkTriangleFilter.h>
 
 #include <pcl/surface/vtk_smoothing/vtk_utils.h>
-
-//#include <pcl/pcl_tests.h>
-
 #include <pcl/point_types.h>
 #include <pcl/sample_consensus/ransac.h>
 #include <pcl/sample_consensus/sac_model_plane.h>
+
+#include <log4cxx/basicconfigurator.h>
+#include <log4cxx/patternlayout.h>
+#include <log4cxx/consoleappender.h>
 #include <tool_path_planner/raster_tool_path_planner.h>
+#include "tool_path_planner/utilities.h"
+
+
+static const std::size_t MAX_ATTEMPTS = 1000;
```

Comment:
Some of these constants assume a particular set of units. For example mm vs meters. Add a comment to state what units are expected. Otherwise, the default thresholds won't work well.